ansible/pn_l2_csv_validation.py

```
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+
+"""
+This python script is to validate L2 VRRP csv file.
+
+Sample CSV file row:
+# VRRP IP, VLAN ID, ACTIVE SWITCH
+101.108.102.0/24, 102, gui-spine1
+
+Example Usage:
+python pn_l2_csv_validation.py l2_csv_file.csv
+
+"""
+
+import csv
+import socket
+import sys
+
+if len(sys.argv) != 2:
+    msg = 'Execution Error: Please provide csv file name!\n'
+    msg += 'Example usage: python pn_l2_csv_validation.py l2_csv_file.csv'
+    exit(msg)
+
+msg = ''
+line_count = 0
+
+reader = csv.reader(open(sys.argv[1]))
+for csv_line in reader:
+    line_count += 1
+    if csv_line[0].startswith('#'):
```

I would suggest:
```
if '#' in csv_line[0]:
```
because the comments line could begin with a space and you dont want to throw an error for that. I see that this is already handled in the l2 vrrp module where spaces are replaced at the beginning itself.